packages/next/src/server/base-server.ts

```diff
@@ -3044,10 +3044,19 @@ export default abstract class Server<
       }
     )
 
-    if (isPrefetchRSCRequest && typeof segmentPrefetchHeader === 'string') {
+    if (isRoutePPREnabled && typeof segmentPrefetchHeader === 'string') {
```

Comment:
woops, this always should have been gated on PPR right?

Reply:
It used to not have the check so that it would 404/204 if PPR was disabled (since why not, nothing should be issuing these requests and it's better than it falling through to a old-style prefetch) but then it turned out I need it to fallthrough to an old-style prefetch anyway.